docs/layouts/shortcodes/apiDeepLink.html

```diff
@@ -0,0 +1,37 @@
+{{- /* apiDeepLink $class $label $pythonFramework */ -}}
+{{- /* Makes a deep link to the currently selected language's documentation for a given class */ -}}
+{{- /* To link to a specific method of the class, supply it as an inner shortcode or string */ -}}
+{{- $class := (.Get 0) -}}
+{{- $label := (.Get 1) -}}
+{{- $pythonFramework := (.Get 2) -}}
```

Comment:
I think making these named params might make it a bit easier to figure out what we're linking to at the call site. Example from another shortcode:

https://github.com/osohq/oso/blob/6df569e81715627444fb260337162ea1eba733ae/docs/layouts/shortcodes/literalInclude.html#L50

Reply:
agree! will do